apps/full-site-editing/full-site-editing-plugin/new-blocks-showcase/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { addFilter } from '@wordpress/hooks';
+import { getCategories, setCategories } from '@wordpress/blocks';
+
+/**
+ * This script needs to run *before* any block is registered! This makes sure
+ * the category filter doesn't miss any blocks targeted for the "New" category.
+ */
+
+const NEW_CATEGORY = {
+	slug: 'new',
+	title: __( 'New', 'a8c' ),
+	// icon: TBD,
```

Comment:
@davemart-in do we have an icon for the "New" category?

Reply:
No. None of the core categories have an icon. I'd vote to just leave it out.